dhis-2/dhis-test-e2e/README.md

```diff
@@ -23,6 +32,16 @@ mvn test \
     -Dtest.track_called_endpoints=true
 ```
 
+### From IntelliJ
+
+Make sure the property instance.url in `dhis-2/dhis-test-e2e/src/main/resources/config.properties`
```

Comment:
This is independent of tomcat 😄 `instance.url` needs to point to DHIS2.

Maybe we need a higher level description at the start of how the e2e tests work. As there are 2 independent aspects 😅 
* how you run DHIS2 (docker, tomcat, embedded jetty?)
* how you run the e2e tests (from intellij, from CLI using maven, from within docker)

Reply:
I'll open a new PR for the readme, thanks!